device/ethernet.yaml

```diff
@@ -31,11 +31,29 @@ components:
       x-include: 
       - '#/components/schemas/Device.EthernetBase'
       description: >-
-        An ethernet interface with an IPv4 and IPv6 interface stack
+        An Ethernet interface with IPv4 and IPv6 addresses.
       type: object
+      required: [port_name]
       properties:
-        ipv4:
-          $ref: './ipv4.yaml#/components/schemas/Device.Ipv4' 
-        ipv6:
-          $ref: './ipv6.yaml#/components/schemas/Device.Ipv6' 
+        port_name: 
+          description: |-
+            The unique name of a Port or LAG object that will contain this 
+            emulated Ethernet interface.
```

Comment:
remove `emulated Ethernet` here

Reply:
since we're not emulating just an `Ethernet` interface here.

Reply:
ok